os/board/common/crashdump.c

```diff
@@ -208,6 +210,65 @@ static int get_ext_userfs_part(size_t *size, int *index)
 	return 0;
 }
 
+static int send_file(void)
+{
+	int i;
+	int fd;
+	int ret;
+	struct stat st;
```

Comment:
How about to initialize values to remove initialization warning in code review tool.